elastic/kibana
# Fix user details risk tab shows when FF is disabled

issue: https://github.com/elastic/kibana/issues/132929

## Summary

The risk tab on the user details page should only show when the FF `riskyUsersEnabled` is enabled.

![Screenshot 2022-05-25 at 15 08 08](https://user-images.githubusercontent.com/1490444/170269420-fe81b299-2e3d-48cf-8e66-2ae0754837ef.png)


### How to test:
* Disabled `riskyUsersEnabled` feature flag
* Open kibana/security
* Open users page
* Click on a user name
* The risk tab should not show



### Checklist
